coderunners/compilers.py

```diff
@@ -11,6 +12,19 @@ class Compiler(ABC):
     def compile(self, submission_path: Path) -> tuple[Path, RunResult]:
         ...
 
+    @classmethod
+    def get_only_file(cls, submission_paths: list[Path], lang: str) -> Path:
+        if len(submission_paths) != 1:
```

Comment:
Let's add a TODO comment that this function should disappear as soon as we implement other compilers correctly